easybuild/easyconfigs/n/NFFT/NFFT-3.5.1-foss-2018b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'NFFT'
+version = '3.5.1'
+
+homepage = 'https://www-user.tu-chemnitz.de/~potts/nfft/'
+description = """The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine
+ library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more
+ dimensions, of arbitrary input size, and of complex data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/NFFT/nfft/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['8d53164d7cd85ad77e1bd03e36c4a99ef73c77f640e527db816cdc3fcb43d6aa']
+
+dependencies = [('FFTW', '3.3.8', '', ('gompi', '2018b'))]
+
+builddependencies = [
+    ('libtool', '2.4.6')
+]
+
+sanity_check_paths = {
+    'files': ['include/nfft3.h', 'include/nfft3mp.h', 'lib/libnfft3.a', 'lib/libnfft3.%s' % SHLIB_EXT],
+    'dirs': ['lib/pkgconfig'],
+}
+
+configure_cmd_prefix = './bootstrap.sh ; '
```

Comment:
Style nitpicking: please move this up above `sanity_check_paths` (we try to maintain an order that matches the order in which steps are executed during the installation process).